packages/client/src/components/DebugDecisionTable.tsx

```diff
@@ -1,32 +1,22 @@
 /* eslint-disable */
 /* istanbul ignore file */
 import { Accordion } from "@amsterdam/asc-ui";
+import { Checker, Question } from "@vergunningcheck/imtr_client";
```

Comment:
Do we need an underscore in imtr_client? This can now be changed to `imtr-client` right?

Reply:
check.